packages/js/product-editor/src/components/editor/editor.tsx

```diff
@@ -37,22 +31,11 @@ import { InterfaceSkeleton } from '@wordpress/interface';
 import { Header } from '../header';
 import { BlockEditor } from '../block-editor';
 import { ValidationProvider } from '../../contexts/validation-context';
-
-export type ProductEditorSettings = Partial<
-	EditorSettings & EditorBlockListSettings
-> & {
-	templates: Record< string, Template[] >;
-};
-
-type EditorProps = {
-	product: Pick< Product, 'id' | 'type' >;
-	productType?: string;
-	settings: ProductEditorSettings | undefined;
-};
+import { EditorProps } from './types';
 
 export function Editor( {
 	product,
-	productType = 'product',
+	postType = 'product',
```

Comment:
You could probably pull this change out into a separate PR that would be pretty small and easy to test. That would keep this PR a bit more focused.

Reply:
reverted